guides/v2.2/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -55,6 +55,9 @@ functional_areas:
     bin/magento cache:clean
     ```
 
+{: .bs-callout-info }
+Not possible to restart the [Elasticsearch](https://www.elastic.co) service itself without Magento's help.
+
```

Comment:
Please, adjust everywhere as “If you need to restart the elasticsearch service, you need to contact Magento support”

Reply:
**done**

Reply:
Thank you for the adjustments. The only place that left unchanged is this one ↑
Please, change this part (lines 58-60) as well. Thank you!

Reply:
duplicated content has been removed . Thanks for notice 👍